modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -1,80 +1,84 @@
-@import 'atlas-variables';
+@import 'cadmin-variables';
 
 $header-height: 64px;
 
-.lfr-layout-reports-panel {
-	&.open-admin-panel.sidenav-menu-slider {
-		visibility: visible;
-		width: 320px;
-	}
-
-	.dropdown-menu__languages {
-		max-height: 295px;
-	}
-
-	.failing-element {
-		&:not(:first-child) {
-			margin-top: 8px;
+html#{$cadmin-selector} {
+	.cadmin {
+		&.lfr-layout-reports-panel {
+			&.open-admin-panel.sidenav-menu-slider {
+				visibility: visible;
+				width: 320px;
+			}
+
+			.dropdown-menu__languages {
+				max-height: 295px;
+			}
+
+			.failing-element {
+				&:not(:first-child) {
+					margin-top: 8px;
+				}
+
+				&:not(:last-child) {
+					border-width: 0 0 1px;
+				}
+
+				border-width: 0;
+			}
+
+			.issue {
+				cursor: pointer;
+			}
+
+			.list-group {
+				margin-bottom: 8px;
```

Comment:
Why are we replacing `rem` with `px`?

Reply:
`rem`'s font size is based on the value of the root element (`html`). If we use `rem` values in cadmin, changing the `font-size` on the `html` element will change the sizing across cadmin.